### ISSUE TITLE
False positive: 'color-contrast' rule fails disabled control's label added using "aria-labelledby" attribute

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#2090]
Same as #1731, but here the label is added using ```aria-labelledby``` attribute

Per WCAG 1.4.3 the text that is part of an inactive user interface component do not have contrast requirement.

> Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

Expectation: 'color-contrast' would not fail for labels related to disabled controls

Actual: 'color-contrast' rule fails for a label( added using ```aria-labelledby``` attribute) of a disabled control

Motivation: Text related to the disabled control is exempt from the contrast requirements

Demo link: https://developer.microsoft.com/en-us/fabric#/controls/web/combobox

![image](https://user-images.githubusercontent.com/22068434/76367465-5ec41700-62ea-11ea-8aa5-332c37d00193.png)

<pre><code>axe-core version: 3.5.1-canary.0ec5091
axe-Coconut version: 4.3.0</code></pre>

**Comment by WilcoFiers** [comment]
Here's another example of this that shouldn't fail axe:

source: https://act-rules.github.io/rules/afw4f7#inapplicable-example-9
```html
<label id="my_pets_name" style="color:#888; background: white;">
	My pet's name
</label>
<div role="textbox" aria-labelledby="my_pets_name" aria-disabled="true"
	style="height:20px; width:100px; border:1px solid black;">
	test
</div>
```

**Comment by padmavemulapati** [comment]
Verified with latest axe (axe-core3.5.5), color-contrast is failing which has no `aria-labelledby` attribute. and its passing which has `aria-labelledby`
Failed-snippet in the  Screenshot 
![image](https://user-images.githubusercontent.com/47409579/87685204-c0b4f180-c7a0-11ea-863c-153afbcc94cd.png)
Pass-snippet in the screenshot:
![image](https://user-images.githubusercontent.com/47409579/87685420-feb21580-c7a0-11ea-8779-47a72535bd77.png)

**Comment by jeankaplansky** [comment]
doc'd in internal release notes on 7/20/2020
